FRW/Foundation/Model/FlowNetworkType.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  FlowNetworkType.swift
+//  FRW
+//
+//  Created by Antonio Bello on 11/20/24.
+//
+
+import Foundation
+import Flow
+import struct SwiftUI.Color
+
+enum FlowNetworkType: String, CaseIterable, Codable {
+    case testnet
+    case mainnet
+    case previewnet
```

Comment:
We can remove previewnet, it's gone

Reply:
Ok removed all references, except for the one defined in the Flow package (it's unmodifiable).
If I knew of the domino effect, I would have deferred this to a separate ticket :-)